lib/server.js

```diff
@@ -175,7 +183,8 @@ function handleRequest (opts, req, resp) {
             query: urlData.query,
             method: req.method.toLowerCase(),
             headers: req.headers,
-            body: body
+            body: body,
+            reqId: reqId
```

Comment:
I was wondering if we could just set the header. That would simplify the logic in the restbase constructor a bit, and would avoid introducing another req member.